library/net.cpp

```diff
@@ -1364,24 +1386,10 @@ int dnet_trans_forward(struct dnet_io_req *r, struct dnet_net_state *orig, struc
 			deadline = request.deadline;
 		}
 
-		if (dnet_time_is_empty(&deadline)) {
+		if (dnet_time_is_empty(&deadline))
```

Comment:
If we have mixed practice of using curly braces and not using after single-line if, I would vote for using braces

Reply:
I asked @shaitan - he voted for keep single lines without braces (the exception: if in function body already occur some cases with other style)